sigstore-java/src/main/java/dev/sigstore/encryption/Keys.java

```diff
@@ -137,9 +139,24 @@ public static PublicKey constructTufPublicKey(byte[] contents, String scheme)
       case "ed25519":
         {
           final KeyFactory kf = KeyFactory.getInstance("Ed25519");
-          final X509EncodedKeySpec keySpec = new X509EncodedKeySpec(contents);
+          X509EncodedKeySpec keySpec;
+          // tuf allows raw keys only for ed25519 (non PEM):
+          // https://github.com/theupdateframework/specification/blob/master/tuf-spec.md#key-objects--file-formats-keys
```

Comment:
I would prefer permalinks rather than links to master branch which might remove or rename file over time

Reply:
Oh right. Yeah I'll fix that